others/src/main/kotlin/app/lawnchair/lawnicons/helper/CommonUtil.kt

```diff
@@ -0,0 +1,71 @@
+package app.lawnchair.lawnicons.helper
+
+import com.google.common.base.Charsets
+import java.io.File
+import java.io.FileWriter
+import java.io.IOException
+import java.nio.file.Path
+import org.apache.commons.io.FilenameUtils
+import org.dom4j.Document
+import org.dom4j.DocumentException
+import org.dom4j.Element
+import org.dom4j.io.OutputFormat
+import org.dom4j.io.SAXReader
+import org.dom4j.io.XMLWriter
+
+class CommonUtil {
+    @Throws(DocumentException::class)
+    fun getElements(
+        document: Document,
+        path: String?,
+    ): List<Element?>? {
+        val rootElement = document.rootElement
+        return rootElement.elements(path)
+    }
+
+    fun sortedMapByValues(map: Map<String, String>): Map<String, String>? {
+        val result2: MutableMap<String, String> = LinkedHashMap()
+        map.entries.stream().sorted(java.util.Map.Entry.comparingByValue())
+            .forEachOrdered { (key, value): Map.Entry<String, String> ->
+                result2[key] = value
+            }
+        return result2
+    }
```

Comment:
Could you please convert these 3 files to be more idiomatic, do not just apply auto-convert results here.

Reply:
I'm not getting you , Did you mean write clean code in kotlin insted of auto-convert from java to kotlin?

Reply:
First you use auto-convert, secondly you should make them more idiomatic and clean.

Reply:
Refactored as much I can.